src/PhpPact/Standalone/StubService/Service/StubServerHttpServiceInterface.php

```diff
@@ -12,5 +12,5 @@ interface StubServerHttpServiceInterface
      *
      * @return string
      */
-    public function getJson(): string;
+    public function getJson(string $endpoint): string;
```

Comment:
A reminder for me to update `UPGRADE-9.0.md` when we are going to review this PR